src/webpack.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { IgnorePlugin } from 'webpack';
```

Comment:
I know :(
Another concern is that we'd be introducing a new API that we must remove once we release the rewrite with adapters...